common/Scripts/helpers.sh

```diff
@@ -1,21 +1,27 @@
 #!/usr/bin/env bash
 
+# shellcheck wants to add '|| exit 1' willy nilly, so let's
+# build a nice 'say' function instead.
+function say() {
+    echo "$*"
```

Comment:
Maybe this should include a `return 1`?

Reply:
Good idea. I like you better than I like ShellCheck it has to be said.